tests/base.py

```diff
@@ -171,8 +171,9 @@ class ImageComparison(object):
     ValueError : at least one of visualizer or ax must be specified.
     """
 
-    def __init__(self, stack, visualizer=None, ax=None, tol=0.01, windows_tol=0.01, ext=".png",
-                 remove_ticks=True, remove_title=True, remove_legend=False):
+    def __init__(self, stack, visualizer=None, ax=None, tol=0.01, 
```

Comment:
I can bump up the tol to 25.5, because it is not a significant change. It seems like it may be due to a difference in colors, because the diff is so extensive. I wonder if we may be using different versions of a package, or if there is some difference in configuration. I ran the tests again locally for Ubuntu with regular Python and conda, and removed the tol=25 to see if it would fail, but it passed in both cases. I'm using matplotlib 3.1.0, and Python 3.6.8.

Sorry for not thinking about the update to the docs, will do!

Reply:
For that flaky test in ubuntu, I agree to bump it up to 25.5. I'm on ubuntu 18.0 ran it on python 3.6.8 and 3.7.3 failing in both. It's hard to say what is causing it without doing a deep dive.

After updating the docs, this should be good to merge in! Thanks again for the great work on this.